CONTRIBUTING.md

```diff
@@ -19,6 +19,16 @@ How to submit a feature request
 - Give a brief explanation about the problem that may currently exist and how your requested feature solves this problem.
 - Try to be as specific as possible. Please not only explain what the feature does, but also how. If your request is about (or includes) changing or extending the UI, describe what the UI would look like and how the user would interact with it.
 
+Testing and Verifying
+--------------------------
+As a developer contributing to AntennaPod, we ask that you test the feature yourself manually and better yet, add unit and functional tests to any feature of bug you fix.
+
+### Running Unit Tests
+* `./gradlew :core:testPlayDebugUnitTest`
+
+### Running integration Tests
+* Start and AVD or plug in your phone
+* `zsh .github/workflows/runTests.sh`
```

Comment:
`zsh` is probably not shipped by default on most systems. Does using `sh` work, too?

Reply:
@ByteHamster could you give me a hint about running a specific integration test? What is the command to do that?

Reply:
I don't use the command line to run tests. Instead, I press the button in Intellij/Android Studio. There you can set up a run configuration like this:


![image](https://user-images.githubusercontent.com/5811634/104302201-d0e82800-54c8-11eb-9e58-c340130a6670.png)

Reply:
thanks

<img width="768" alt="antennapod-run-tests" src="https://user-images.githubusercontent.com/149837/105122859-e1317180-5a8b-11eb-8d45-d54a3b051a9b.png">